src/scripts/docker/build-wheel-linux.sh

```diff
@@ -30,42 +33,69 @@ export BITWUZLA_LIBRARIES=$DEPENDENCIES_DIR/bitwuzla/install/lib64/libbitwuzla.s
+#export PYTHON_LIBRARY=$($PYTHON_BINARY -c "from sysconfig import get_paths; print(get_paths()['include'])")
+conda activate py38
+python -m pip install build
+python --version
+python -m build --wheel --outdir $WHEEL_DIR/linux_x86_64
 
 # Build Triton Python wheel package for Python 3.9.
 echo "[+] Build Triton wheel package for Python 3.9"
 cd $SOURCE_DIR
-export PYTHON_BINARY=/opt/_internal/cpython-3.9.17/bin/python
-export PYTHON_INCLUDE_DIRS=$($PYTHON_BINARY -c "from sysconfig import get_paths; print(get_paths()['include'])")
-export PYTHON_LIBRARY=$($PYTHON_BINARY -c "from sysconfig import get_paths; print(get_paths()['include'])")
-
-$PYTHON_BINARY -m build --wheel --outdir $WHEEL_DIR/linux_x86_64
+#export PYTHON_BINARY=/opt/_internal/cpython-3.9.17/bin/python
+#export PYTHON_INCLUDE_DIRS=$($PYTHON_BINARY -c "from sysconfig import get_paths; print(get_paths()['include'])")
+#export PYTHON_LIBRARY=$($PYTHON_BINARY -c "from sysconfig import get_paths; print(get_paths()['include'])")
+conda activate py39
+python -m pip install build
+python --version
+python -m build --wheel --outdir $WHEEL_DIR/linux_x86_64
 
 # Build Triton Python wheel package for Python 3.10.
 echo "[+] Build Triton wheel package for Python 3.10"
 cd $SOURCE_DIR
-export PYTHON_BINARY=/opt/_internal/cpython-3.10.12/bin/python
-export PYTHON_INCLUDE_DIRS=$($PYTHON_BINARY -c "from sysconfig import get_paths; print(get_paths()['include'])")
-export PYTHON_LIBRARY=$($PYTHON_BINARY -c "from sysconfig import get_paths; print(get_paths()['include'])")
-
-$PYTHON_BINARY -m build --wheel --outdir $WHEEL_DIR/linux_x86_64
+#export PYTHON_BINARY=/opt/_internal/cpython-3.10.12/bin/python
+#export PYTHON_INCLUDE_DIRS=$($PYTHON_BINARY -c "from sysconfig import get_paths; print(get_paths()['include'])")
+#export PYTHON_LIBRARY=$($PYTHON_BINARY -c "from sysconfig import get_paths; print(get_paths()['include'])")
+conda activate py310
+python -m pip install build
+python --version
+python -m build --wheel --outdir $WHEEL_DIR/linux_x86_64
 
 # Build Triton Python wheel package for Python 3.11.
 echo "[+] Build Triton wheel package for Python 3.11"
 cd $SOURCE_DIR
-export PYTHON_BINARY=/opt/_internal/cpython-3.11.4/bin/python
-export PYTHON_INCLUDE_DIRS=$($PYTHON_BINARY -c "from sysconfig import get_paths; print(get_paths()['include'])")
-export PYTHON_LIBRARY=$($PYTHON_BINARY -c "from sysconfig import get_paths; print(get_paths()['include'])")
-
-$PYTHON_BINARY -m build --wheel --outdir $WHEEL_DIR/linux_x86_64
+#export PYTHON_BINARY=/opt/_internal/cpython-3.11.4/bin/python
+#export PYTHON_INCLUDE_DIRS=$($PYTHON_BINARY -c "from sysconfig import get_paths; print(get_paths()['include'])")
+#export PYTHON_LIBRARY=$($PYTHON_BINARY -c "from sysconfig import get_paths; print(get_paths()['include'])")
+conda activate py311
+python -m pip install build
+python --version
+python -m build --wheel --outdir $WHEEL_DIR/linux_x86_64
 
+python -m pip install auditwheel
```

Comment:
@cnheitman is this update ok for you? (as I know you worked a lot on this file too).